net-libs/ccrtp/ccrtp-2.1.2.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="GNU ccRTP - Implementation of the IETF real-time transport protocol"
+HOMEPAGE="https://www.gnu.org/software/ccrtp/"
+SRC_URI="mirror://gnu/ccrtp/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0/2"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug demos doc static-libs"
+
+RDEPEND="
+	>=dev-libs/libgcrypt-1.2.3
+	>=dev-libs/ucommon-6.3.1[cxx]"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )"
+
+src_prepare() {
+	default
+
+	if ! use doc; then
+		sed -i -e '/^SUBDIRS.*=/s#doc##' Makefile.am \
+			-e 's#^doc/Makefile.##' configure.ac || die "sed failed"
+	fi
+
+	eautoreconf
+}
+
+src_configure() {
+	econf --enable-static=$(usex static-libs) $(use_enable debug) $(use_enable demos)
+}
+
+src_install() {
+	default
+
+	find "${ED}" -name '*.la' -type f -delete || die
+
+	if use demos; then
+		dodir "/usr/share/${PN}"
+		dodir "/usr/share/${PN}/.libs"
+		find "${S}/demo" -exec test -x {} \; -exec cp {} "${D}/usr/share/${PN}" \;
+		find "${S}/demo/.libs" -exec test -x {} \; -exec cp {} "${D}/usr/share/${PN}/.libs" \;
```

Comment:
If they are binaries compiled for a certain arch, they don't belong to /usr/share.

What's the use of installing these demos at all?

Reply:
How is the best practise? I wanted to leave out first. But it's a functionality on the part of the package that the Gentoo ebuild should map. Or?

Reply:
If you think they can be useful to the end user, I think it's fine to install them. But to `/usr/$(get_libdir)` if they are binaries instead of /usr/share.

But as upstream's `make install` doesn't install them, probably they are not intended to be installed at all.